.github/List.jsx

```diff
@@ -0,0 +1,79 @@
+import { useState } from 'react';
```

Comment:
seems like `List.jsx` was accidentally copied and pasted into the `.github` folder? Seems like it still exists under `views`.